omod-common/src/main/java/org/openmrs/module/webservices/docs/swagger/SwaggerSpecificationCreator.java

```diff
@@ -1037,7 +1037,10 @@ private Operation createOperation(DelegatingResourceHandler<?> resourceHandler,
 		        .consumes("application/json").produces("application/json");
 		
 		// create definition
-		if (operationName == "post" || operationName == "get") {
+
+		if (operationName.equals("post") || operationName.equals("get")) {
+			//			createDefinition(operationEnum, resourceName, resourceParentName, representation);
+
 			createDefinition(operationEnum, resourceName, resourceParentName, resourceHandler);
```

Comment:
```suggestion
		if (operationName.equals("post") || operationName.equals("get")) {
			//			createDefinition(operationEnum, resourceName, resourceParentName, representation);
			createDefinition(operationEnum, resourceName, resourceParentName, resourceHandler);
```
Don't you think these blank lines are unnecessary?

Reply:
And is this code in the comment needed?